lisp/ess-inf.el

```diff
@@ -149,33 +146,33 @@ Alternatively, it can appear in its own frame if
              (eq major-mode 'inferior-ess-mode))
         (setq startdir  (if ess-ask-for-ess-directory
                             (ess-get-directory defdir temp-dialect procname)
-                          defdir)
-              buf       (current-buffer)
-              ;; don't change existing buffer name in this case; It is very
-              ;; commong to restart the process in the same buffer.
-              buf-name-str (buffer-name)
-              method    1))
+                          defdir))
```

Comment:
Why over the sudden you decided for this? I aggregate setqs whenever I see the chance ;) Find it neater and more readable. But well, it's tiny.

Reply:
I thought aggregated setq were against coding standards but after a bit of googling it seems I was mistaken and there are two standard visions of two groups of people arguing over this ;)

Personally I prefer disaggregated because it attracts attention over small infelicities (mutation), are easier to refactor, and I personally find it more readable. I can switch it back to previous form if you prefer.

Reply:
Nevermind. It's not important. Not what we have some high standards in this project for coding style:). But in my code I always aggregate as I find all those side-effectful repeated `setq` noisy and annoying.